it/src/test/java/org/corfudb/universe/scenario/OneNodeDownIT.java

```diff
@@ -44,15 +45,14 @@ public void oneNodeDownTest() {
             }
 
             testCase.it("Should stop one node and then restart", data -> {
-                CorfuServer server0 = corfuCluster.getFirstServer();
+                CorfuServer server2 = corfuCluster.getServerByIndex(2);
```

Comment:
Why changing the servers?

Reply:
Because servers are already sorted by their endpoints. If we take down server0, with the new algorithm, server2 could first become unresponsive because of a timing issue (server1 has a stale heartbeat before server0 down, thus sending this view to server2, so server2 think the link between server0 and server1 is health, but server2 itself has a new heartbeat and knows about server0 down. So server2 make decision locally and mark itself unresponsive since it has larger endpoint).

Using server2 here make test stable because we  make sure the node we taken down is the node becomes unresponsive later.

Reply:
However once we introduce the stabilizer, we can safely take down any node, that's why I put a TODO below.